faculty/clients/experiment.py

```diff
@@ -293,3 +340,40 @@ def list_runs(
         return self._get(
             endpoint, ListExperimentRunsResponseSchema(), params=query_params
         )
+
+    def log_run_data(
+        self, project_id, run_id, metrics=None, params=None, tags=None
+    ):
+        """Update the data of a run.
+
+        Parameters
+        ----------
+        project_id : uuid.UUID
+        run_id : uuid.UUID
+        metrics : List[Metric], optional
+            Each metric will be inserted.
+        params : List[Param], optional
+            Each param will be inserted. Note that on a name conflict the
+            entire operation will be rejected.
+        tags : List[Tag], optional
+            Each tag be upserted.
+
+        Returns
+        -------
+        None
```

Comment:
I think that typically in this case you'd just omit the `Returns` block completely since we're kind of not returning anything.

Reply:
👍

Reply:
👍

Reply:
👍